### PR TITLE
Forms: View / Edit Mode - Conceptual Design

### PR DESCRIPTION
**Pull request by zhutaoredhat**
## Description
Provide a design recommendation on how to address view / edit mode in a form.
The JIRA issue: https://patternfly.atlassian.net/browse/PTNFLY-1746

## Changes

Write a list of changes the PR introduces

* Added the documents (overview and design description).
* Added the design files.

## New Pattern Checklist (if relevant)

- [x] Pattern does not already exist
- [x] Pattern solves common problems
- [ ] Pattern has community interest (You've addressed the PatternFly mailing list)
- [x] New files are in a folder titled with the name of the pattern
- [x] All images are in an "img" folder and embedded images have the correct filepath
- [x] Both overview.md and design.md files are present

### CODE REVIEW

File: pattern-library/forms-and-controls/view-edit-mode/design.md [hunk 1]
```diff
@@ -4,7 +4,6 @@
 ![Image highlighting attribute selector](img/View.png)
 
  - The "Edit" button has been put in the top right corner. Horizontal center align with the title.
```
Comment (LHinson) [inline pattern-library/forms-and-controls/view-edit-mode/design.md]: These should be numbered (instead of bulleted) to match annotated notes

Comment (zhutaoredhat) [inline pattern-library/forms-and-controls/view-edit-mode/design.md]: Hi @LHinson ，Thanks for your quick feedback. The examples is comes form https://docs.google.com/document/d/1qyedqGW1LQkvXyf44loTv25qZ6cPiPIRYKcOdhCM4PY/edit# page 11. I will update them soon.

**Comment by mcarrano** [comment]
@zhutaoredhat I finally had a chance to look at this.  In general the design makes sense, however I see that the mockups are different between the Overview and Design pages, so not sure which version is correct.  I also wonder about the placement of the Edit button.  I actually like this next to the form title as I fear it can get lost if it's right aligned on a page that is stretched very wide.  But that said, I also wonder if we considered using the "pencil" edit icon here instead of a text button as having the standard button to the right of the title feels a bit heavy.  Thoughts about this?

**Comment by zhutaoredhat** [comment]
@mcarrano Hi Matt, Thanks for your feedback. That is great helpful to me. I have collected some examples about this task and leave some comments in the end. https://docs.google.com/document/d/1WnyRyCiyQsmRxWuDonAS2HJYZeSLr9k6f_gvNmoGU1U/edit#
Most “Edit” button in patternfly project in on the right top corner, so I want keep this usage habits.
The pencil icon I really love it. I had put it in the edit button looks much more clearly.

**Comment by mcarrano** [comment]
@zhutaoredhat and @LHinson This pull request is being closed in favor of a new editable forms guidelines page that will be written.

**Comment by zhutaoredhat** [comment]
@mcarrano Sure. Thanks~

**Pull request closed without merging**